bucoffea/plot/stack_plot.py

```diff
@@ -183,14 +201,24 @@ def make_plot(acc, region, distribution, year,  data, mc, outdir='./output/stack
     ax.legend(title=region_name,ncol=1)
 
     # Ratio plot
-    hist.plotratio(h[data].integrate('dataset'), h[mc].integrate('dataset'),
-                ax=rax,
-                denom_fill_opts={},
-                guide_opts={},
-                unc='num',
-                overflow='all',
-                error_opts=data_err_opts
-                )
+    if data is not None:
+        hist.plotratio(h[data].integrate('dataset'), h[mc].integrate('dataset'),
+                    ax=rax,
+                    denom_fill_opts={},
+                    guide_opts={},
+                    unc='num',
+                    overflow='all',
+                    error_opts=data_err_opts
+                    )
+    if signal is not None:
```

Comment:
This is not very useful at the moment as signal is usually << background, so the ratio plot will not be very helpful